lib/api/unittest/CSingleStreamDataAdderTest.cc

```diff
@@ -73,42 +73,42 @@ void detectorPersistHelper(const std::string& configFileName,
 
     std::string origSnapshotId;
     std::size_t numOrigDocs(0);
-    ml::api::CAnomalyJob origJob(
-        JOB_ID, limits, fieldConfig, modelConfig, wrappedOutputStream,
-        std::bind(&reportPersistComplete, std::placeholders::_1,
-                  std::ref(origSnapshotId), std::ref(numOrigDocs)),
-        nullptr, -1, "time", timeFormat);
```

Comment:
It looks like there are masses of changes in this file but most are whitespace differences caused by me moving braces to adjust object lifetimes.  I did this after changing the resource monitor to reduce memory consumption on unregistering a component.  This test was relying on the old (unintuitive in my opinion) behaviour.

Best to review this file with "ignore whitespace differences" enabled!

Reply:
👍